tests/plugins/dialog/_helpers/tools.js

```diff
@@ -0,0 +1,322 @@
+								}
+							]
+						},
+						{
+							id: 'mp-test2',
+							label: 'MP 2',
+							elements: [
+								{
+									type: 'text',
+									id: 'mp-input21',
+									label: 'input 21'
+								}
+							]
+						},
+						{
+							id: 'mp-test3',
+							label: 'MP 3',
+							elements: [
+								{
+									type: 'text',
+									id: 'mp-input31',
+									label: 'input 31'
+								},
+								{
+									type: 'text',
+									id: 'mp-input32',
+									label: 'input 32'
+								},
+								{
+									type: 'text',
+									id: 'mp-input33',
+									label: 'input 33'
+								},
+								{
+									type: 'text',
+									id: 'mp-input34',
+									label: 'input 34'
+								},
+								{
+									type: 'text',
+									id: 'mp-input35',
+									label: 'input 35'
+								}
+							]
+						}
+					],
+					onLoad: onLoadHandler
+				};
+			},
+			hiddenPage: function() {
+				return {
+					title: 'Hidden page dialog',
+					contents: [
+						{
+							id: 'hp-test1',
+							label: 'HP 1',
+							elements: [
+								{
+									type: 'text',
+									id: 'hp-input11',
+									label: 'input 11'
+								},
+								{
+									type: 'text',
+									id: 'hp-input12',
+									label: 'input 12'
+								},
+								{
+									type: 'text',
+									id: 'hp-input13',
+									label: 'input 13'
+								}
+							]
+						},
+						{
+							id: 'hp-test2',
+							label: 'HP 2',
+							elements: [
+								{
+									type: 'text',
+									id: 'hp-input21',
+									label: 'input 21',
+									requiredContent: 'fakeElement' // This input element should be hidden.
+								},
+								{
+									type: 'text',
+									id: 'hp-input22',
+									label: 'input 22'
+								},
+								{
+									type: 'text',
+									id: 'hp-input23',
+									label: 'input 23',
+									requiredContent: 'fakeElement' // This input element should be hidden.
+								}
+							]
+						},
+						{
+							id: 'hp-test3',
+							label: 'HP 3',
+							requiredContent: 'fakeElement', // Entire tab should be hidden.
+							elements: [
+								{
+									type: 'text',
+									id: 'hp-input31',
+									label: 'input 31'
+								},
+								{
+									type: 'text',
+									id: 'hp-input32',
+									label: 'input 32'
+								},
+								{
+									type: 'text',
+									id: 'hp-input33',
+									label: 'input 33'
+								},
+								{
+									type: 'text',
+									id: 'hp-input34',
+									label: 'input 34'
+								},
+								{
+									type: 'text',
+									id: 'hp-input35',
+									label: 'input 35'
+								}
+							]
+						},
+						{
+							id: 'hp-test4',
+							label: 'HP 4',
+							elements: [
+								{
+									type: 'text',
+									id: 'hp-input41',
+									label: 'input 41'
+								}
+							]
+						}
+					],
+					onLoad: onLoadHandler
+				};
+			}
+		},
+
+		// Assert for checking dialog elements and buttons.
+		// config.buttonName {String} 'ok' or 'cancel'
```

Comment:
Even though this docs will no be parsed it would be good to follow official format so:

```
@param config {Object}
@param config.buttonName {String} Button name, usually 'ok' or 'cancel'.
```

I would also state that you can pass `buttonName` or `tab` and `elementId` so button or element will be focused and that it can't be used both at the same time.